src/components/Shared/HeaderBanner/HeaderBanner.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import React, { useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons'
+import backgroundImage from '../../../images/github_pattern.svg'
+import './HeaderBanner.css'
+
+type ColorScheme = {
+    primary: string,
+    secondary: string
+};
+
+type Props = {
+    title: string,
+    mainText: Array<string>,
+    subText: Array<string>,
+    bgColorScheme: ColorScheme,
+    iconColorScheme: ColorScheme
+};
+
+function HeaderBanner({
```

Comment:
nice, maybe we should change some classes that don't do much to functional components

Reply:
+1